src/platform/graphics/graphics-device.js

```diff
@@ -39,6 +39,13 @@ class GraphicsDevice extends EventHandler {
      */
     isWebGPU = false;
 
+    /**
```

Comment:
I'd probably remove this public API. You can detect what device you have by:
device.isWebGPU: webgpu
device.webgl2: webgl2
otherwise: webgl1

Reply:
actually, thinking about this, maybe leave it in, it's definitely more convenient. I added it a while back removed in favour is easier to use device.isWebGPU (does not need any import, compared to `device.deviceType === some-constant`) but for some cases this is easier to have.